CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributing to GenLayer Simulator
+
+We're thrilled that you're interested in working on the GenLayer Simulator! This document will guide you through the contribution process.
+
+## What is the GenLayer Simulator?
+
+The GenLayer Simulator is an interactive sandbox designed for developers to explore the potential of GenLayer's Intelligent Contracts. It replicates the GenLayer network's execution environment and consensus algorithm, providing a controlled and local environment to test different ideas and behaviors.
+
+Begin by setting up your development environment to start contributing: Follow our [README](https://github.com/yeagerai/genlayer-simulator/blob/main/README.md) for detailed setup instructions.
+
+## How You Can Contribute?
+
+Contributions to the GenLayer Simulator are welcome in several forms:
+
+### Testing the Simulator and Providing Feedback
+
+Help us enhance user experience and system functionality.
+- **Feature Testing**: Provide feedback on new features in our [Discord channel](https://discord.gg/5Gq4nCxe).
```

Comment:
We also have a feedback from https://docs.google.com/forms/d/1IVNsZwm936kSNCiXmlAP8bgJnbik7Bqaoc3I6UYhr-o/viewform?edit_requested=true